crates/ursa-rpc-service/src/api.rs

```diff
@@ -96,6 +106,12 @@ pub trait NetworkInterface: Sync + Send + 'static {
 
     /// Get the addresses that p2p node is listening on
     async fn get_listener_addresses(&self) -> Result<Vec<Multiaddr>>;
+
+    ///Stream txn to the Narwhal worker mempool
+    async fn submit_narwhal_txn(&self, txn: TransactionProto) -> Result<()>;
+
+    ///Query the application layer through abci
+    async fn query_abci(&self, txn: AbciQueryQuery) -> Result<ResponseQuery>;
 }
 
 type PendingRequests = Arc<RwLock<HashMap<Cid, Vec<Sender<Result<u64>>>>>>;
```

Comment:
Is it safe to use `UnboundedSender` here?

Reply:
Id like to keep this bounded. I also think we should eventually refactor the rest of this file to also use bounded once we work at some sane limits. Narwhal gets the other end of this and they use bounded everywhere. If you feel strongly about this im open to changing

Reply:
`UnboundedSender` is renamed to `Sender` https://github.com/fleek-network/ursa/pull/428/files#diff-b3908f39034f4974e06b64ca4ec45dd178be3b9eabd3c4e393c4433d890dd787R27 so that's why i was wondering about the safety of using unbounded here. I agree we should use bounded.

Reply:
This line wasnt part of the task/PR though so feel free to ignore.